distribution/src/bin/elasticsearch-cli

```diff
@@ -2,6 +2,8 @@
 
 set -e -o pipefail
 
+# set tempdir to non empty string so env script does not create it, as it is not used here
+ES_TMPDIR="notused"
```

Comment:
This feels like a hack. Even if we proceed with this, a test would be nice because we're otherwise maintaining a hack without any tests. A non-hack would be ideal though. 😇

Reply:
I could alternatively just move the temp dir creation out of env and into bin/ES scripts? I don't have any good ideas for a test. The directory created by mktemp and equivalents is often obfuscated so I can't figure out if one was created.